Tutorial.md

```diff
@@ -0,0 +1,178 @@
+# Welcome to the BiobakeryUtils tutorial
+
+## Using BiobakeryUtils with [MetaPhlAn](https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-3.0) files
+
+### Input file format
+
+BiobakeryUtils accepts `.tsv` files outputted from MetaPhlAn.
+Files can either contain a merged table or one or multiple single table(s).
```

Comment:
Maybe mention that this comes from the `metaphlan_merge_tables` command, and link to the relevant MetaPhlAn docs?

Reply:
Is this the one you mean: https://github.com/biobakery/MetaPhlAn/blob/master/metaphlan/utils/merge_metaphlan_tables.py ?